app/js/index.js

```diff
@@ -11,7 +11,7 @@ window.webslack = (function (gui, urllib, pkg, localStorage) {
 
 	var win = gui.Window.get();
 	var validSlackSubdomain = /(.+)\.slack.com/i;
-	var validSlackRedirect = /(.+\.)?slack-redir.com/i;
+	var validSlackRedirect = /(.+\.)?slack-redir.net/i;
```

Comment:
@canuc Any reason why we have a URL check for new windows? Why not open all of them as is?